.github/workflows/release.yml

```diff
@@ -0,0 +1,34 @@
+name: Create Release
+on:
+    workflow_dispatch:
```

Comment:
perhaps this could happen automatically on pushing tags as well? Then we could probably name the releases after something more human-friendly than just a commit hash :)

Reply:
There's [on.push.tags](https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#onpushbranchestagsbranches-ignoretags-ignore)

I believe this should work, but I haven't tested it
```
on:
  push:
    tags:
      - '*'
```